api/src/main/java/io/minio/MinioClient.java

```diff
@@ -199,7 +199,7 @@
 
   private String userAgent = DEFAULT_USER_AGENT;
 
-  private OkHttpClient httpClient = new OkHttpClient();
+  private OkHttpClient httpClient;
```

Comment:
Shouldn't `httpClient` be initialized in all other constructors (other than `MinioClient(String endpoint, int port, String accessKey, String secretKey, String region, boolean secure, OkHttpClient httpClient)` as well?

Reply:
Not required. Because all constructors call this constructor only.